### PR TITLE
installation in lumen

### PR DESCRIPTION
**Pull request by SOSTheBlack**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,9 +36,17 @@ Then run a composer update
 php composer.phar update
 ```
 
-To use the AWS Service Provider, you must register the provider when bootstrapping your Laravel application.
+To use the AWS Service Provider, you must register the provider when bootstrapping your application.
 
-Find the `providers` key in your `config/app.php` and register the AWS Service Provider.
+#### Lumen
+In Lumen find the `Register Service Providers` in your `bootstrap/app.php` and register the AWS Service Provider.
```
Comment (cjyclaire) [inline README.md]: Could you add a new line there?

Comment (cjyclaire) [inline README.md]: A newline there as well?

**Comment by SOSTheBlack** [comment]
Still adding new lines, github removes new lines

**Comment by cjyclaire** [comment]
@SOSTheBlack Now the changes looks good, will merge it once CI completes :)

File: README.md [hunk 2]
```diff
@@ -36,9 +36,18 @@ Then run a composer update
...
 ```
 
-To use the AWS Service Provider, you must register the provider when bootstrapping your Laravel application.
+To use the AWS Service Provider, you must register the provider when bootstrapping your application.
 
-Find the `providers` key in your `config/app.php` and register the AWS Service Provider.
+
+### Lumen
+In Lumen find the `Register Service Providers` in your `bootstrap/app.php` and register the AWS Service Provider.
+```php
```
Comment (cjyclaire) [inline README.md]: Apology for mini pick on this, yet, could you add new line before the code section and remove the one extra new line after the code section?

**Pull request merged**